doc/userguide/capture-hardware/ebpf-xdp.rst

```diff
@@ -350,11 +375,160 @@ Confirm you have the XDP filter engaged in the output (example)::
  ...
  ...
 
+Pinned maps usage
+-----------------
+
+Pinned maps stay attached to the system if the creating process disappears and
+they can also be accessed by external tools. In Suricata bypass case, this can be
+used to keep active bypassed flow tables so Suricata is not hit by previsouly bypassed flows when
+restarting. In the socket filter case, this can be used to maintain a map from tools outside
+of Suricata.
+
+To used pinned maps, you first have to mount the `bpf` pseudo filesystem ::
+
+  sudo mount -t bpf none /sys/fs/bpf
+
+You can also add to you `/etc/fstab` ::
+
+ bpffs                      /sys/fs/bpf             bpf     defaults 0 0
+
+and run `sudo mount -a`.
+
+Pinned maps will be accessible as file from the `/sys/fs/bpf` directory. Suricata
+will pin them under the name `suricata-$IFACE_NAME-$MAP_NAME`.
+
+To activate pinned maps for a interface, set `pinned-maps` to `true` in the `af-packet`
+configuration of this interface ::
+
+  - interface: eth3
+    pinned-maps: true
+
+This option can be used to expose the maps of a socket filter to other processes.
+This allows for example, the external handling of a blacklist or white list of
```

Comment:
whitelist

Reply:
Switching to politically correct with accept list and block list.